sketch_map_tool/analyses/pdf_gen/pdf_gen.py

```diff
@@ -5,32 +5,36 @@
 """
 # pylint: disable=too-many-locals,invalid-name
 
-import os
 import json
-from typing import List, Tuple, Optional
+import os
+from time import strftime
+from typing import List, Optional, Tuple
 
-from reportlab.platypus import Paragraph
-from reportlab.pdfgen import canvas
 from reportlab.lib import utils
 from reportlab.lib.pagesizes import A4
-from reportlab.lib.units import mm
 from reportlab.lib.styles import ParagraphStyle
+from reportlab.lib.units import mm
 from reportlab.pdfbase.pdfmetrics import stringWidth
-from time import strftime
+from reportlab.pdfgen import canvas
+from reportlab.platypus import Paragraph
 
 from sketch_map_tool.analyses.helpers import AnalysisResult, QualityLevel
-from sketch_map_tool.analyses.html_gen.analyses_output_generator import get_general_score
+from sketch_map_tool.analyses.html_gen.analyses_output_generator import (
```

Comment:
This line previously had 74 characters, so why adding this linebreak and unnecessary brackets? Does the autoformatter perhaps have some very short line length setting?

Reply:
The line has 89 character thas why it got changed by black/isort.

Reply:
Oh wait. Have to check that.

Reply:
Black formats lines to be below 88 characters. Isort as well.